README.md

```diff
@@ -39,11 +39,21 @@ Compared with other graph database solutions, **Nebula Graph** has the following
 * SQL-like query language
 * Role-based access control for higher level security
 
+## About this Repo
+
+Please note that this repo is only for Nebula Graph `v1.*.*`.
+
+From `v2.*.*`, which was already GA in 2021 April, Nebula Graph's core code bases are [nebula-graph](https://github.com/vesoft-inc/nebula-graph), [nebula-common](https://github.com/vesoft-inc/nebula-common) and [nebula-storage](https://github.com/vesoft-inc/nebula-storage), it's recommended to start from the repo: [nebula-graph](https://github.com/vesoft-inc/nebula-graph).
```

Comment:
the GA version of which has already been released in April 2021,

Codebase is more popular to programmers than code base.

Reply:
done